### ISSUE TITLE
Alien Operating Tables don't get numbing

### ISSUE THREAD

**Issue by unit0016** [effigy-se/effigy-se#411]
See title. Something related to a proc somewhere.

RANT TIMEEEE

Numbing as it is right now is fucking stupid and redundant anyways. I'm sorry, man, but there is ZERO effective change from this stupid half-assed attempt to get people to actually do medRP. Every time we've have surgery done so far its been either:

On an optable
On stasis

Period.
The mood debuff for this is high negligible anyways, AND punishes the guy who isn't making the mistake in the first place.

Tl;Dr, go all the way or don't go at all. This is stupid.
